clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/ProtocolResourceDefinition.java

```diff
@@ -278,6 +278,12 @@ public void registerAttributes(ManagementResourceRegistration registration) {
         }
     }
 
+    @Override
+    public void registerCapabilities(ManagementResourceRegistration resourceRegistration) {
+        super.registerCapabilities(resourceRegistration);
+        resourceRegistration.registerCapability(Capability.SOCKET_BINDING.definition);
+    }
+
```

Comment:
This capability is already registered with the resource. See:
https://github.com/wildfly/wildfly/blob/master/clustering/common/src/main/java/org/jboss/as/clustering/controller/AddStepHandler.java#L103